apps/sms/views/conversation/js/conversation.js

```diff
@@ -513,6 +514,9 @@ var ConversationView = {
     // TODO should we implement hooks to Navigation so that Threads could
     // get an event whenever the panel changes?
     Threads.currentId = args.id;
+    this.currentThreadId = args.id;
+    // TODO: Will need to replace with Conversation service method.
+    this.currentThread = Threads.active;
```

Comment:
and wondering if we need `currentThreadId`, as maybe `this.activeThread.id` works just as fine ?